dataflow/flex-templates/streaming_beam.py

```diff
@@ -0,0 +1,145 @@
+#
```

Comment:
It looks like top line is missing. Could you copy it from a different file?

Reply:
Hi, Thanks for the comment.
I have added the README and other setup files for the same.  I have also addressed your comment.
This sample is similar to what David is writing for java here (https://github.com/davidcavazos/java-docs-samples/tree/dataflow-flex-templates/dataflow/flex-templates/streaming_beam_sql). This is required for NEXT where we want to have sample python example for DataFlow flex templates.